macros/macros.in

```diff
@@ -36,6 +36,7 @@
 %__echo			/bin/echo
 %__fakeroot		/usr/bin/fakeroot
 %__find			/usr/bin/find
+%__gem         /usr/bin/gem
```

Comment:
Can you please tab align this like the others?

Reply:
Yup, didn't notice that this file is tab-aligned. Probably should tweak my editor to highlight that, thanks.